libs/portals/my-pages/health/src/screens/Vaccinations/VaccinationsWrapper.tsx

```
@@ -44,25 +44,21 @@ export const VaccinationsWrapper = () => {
-        />
-        <Box marginLeft={1}>
-          <LinkButton
-            to={formatMessage(m.makeVaccinationAppointmentLink)}
-            icon="open"
-            variant="utility"
-            text={formatMessage(m.makeVaccinationAppointment)}
-          />
-        </Box>
-      </Box>
-
+        />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)